rhods-operator/instance/components/components-serving/patch-datasciencecluster.yaml

```diff
@@ -1,11 +1,12 @@
 kind: DataScienceCluster
 apiVersion: datasciencecluster.opendatahub.io/v1
 metadata:
-  name: default
+  name: default-dsc
 spec:
   components:
     kserve:
       managementState: Managed
+      defaultDeploymentMode: Serverless
```

Comment:
Will this break with older versions?

It looks like it is an optional field in 2.13 so I think it should be fine there.  Do you know if the field is present in 2.10?

Reply:
this flag was actually added in 2.8, 
by default if not specify in DSC, Serverless will be used than RawDeployment (the other option). User might not be aware of this behind the scenes. 
i can drop this change if adding it can cause more confusion than clarification.